digits/dataset/tasks/create_db.py

```diff
@@ -301,24 +302,60 @@ def distribution_data(self):
         if len(self.distribution.keys()) != len(labels):
             return None
 
-        values = ['Count']
+        label_count = 'Count'
+        label_error = 'LoadImageError'
+
+        # backwards compatibility
+        if not hasattr(self, 'entries_error'):
+            self.entries_error = 0
+            for key in self.distribution.keys():
+                self.distribution[key] = {
+                    'count': self.distribution[key],
+                    'error_count': 0
+                }
```

Comment:
For consistency, it's better to do this kind of backwards-compatibility upgrade in the `__setstate__` function above